src/libraries/Microsoft.Extensions.Configuration.Abstractions/ref/Microsoft.Extensions.Configuration.Abstractions.cs

```diff
@@ -13,6 +13,7 @@ public static partial class ConfigurationExtensions
         public static System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string, string>> AsEnumerable(this Microsoft.Extensions.Configuration.IConfiguration configuration, bool makePathsRelative) { throw null; }
         public static bool Exists(this Microsoft.Extensions.Configuration.IConfigurationSection section) { throw null; }
         public static string GetConnectionString(this Microsoft.Extensions.Configuration.IConfiguration configuration, string name) { throw null; }
+        public static IConfigurationSection GetRequiredSection(this IConfiguration configuration, string key) { throw null; }
     }
```

Comment:
This diff needs to get auto-generated:
You'll need to follow the instruction for updating ref source under [For most assemblies within libraries](https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/updating-ref-source.md)

Reply:
I fixed that thanks for the link to the documentation :)